docs/docs/_clisettings.mdx

```diff
@@ -10,6 +10,7 @@ Setting name|Definition|Default value
 `disableTelemetry`|Disables sending of telemetry data|`false`
 `errorOutput`|Defines if errors should be written to `stdout` or `stderr`|`stderr`
 `helpMode`|Defines what part of command's help to display. Allowed values are `options`, `examples`, `remarks`, `response`, `full`|`full`
+`helpTarget`|Defines the way the command help will be shown. Allowed values are `console` or `online`
```

Comment:
We are missing the default value here.
Also: `online` should be `web` according to the specs.